Tabuada/Tabuada/script.js

```diff
@@ -0,0 +1,37 @@
+const button = document.getElementById("subBtn");
+const input = document.getElementById("multiplierInput");
+const form = document.getElementById("multiplierForm");
+const table = document.getElementById("tabuadaTable") 
+
+let number = "";
+
+form.addEventListener("submit",(e)=>{
+    e.preventDefault();
+})
+
+button.addEventListener("click",()=>{
+    number = input.value; 
+    table.innerHTML = null;
```

Comment:
```suggestion
    table.innerHTML = "";
```
Funciona mas deve-se colocar alguma coisa e não mudar para null